etc/c/odbc/sql.d

```diff
@@ -1,16 +1,437 @@
+enum SQL_DESC_ALLOC_USER=2;
+enum SQL_DESC_ALLOC_TYPE=1099;
+enum SQL_DESC_COUNT=1001;
+enum SQL_DESC_TYPE=1002;
+enum SQL_DESC_LENGTH=1003;
+enum SQL_DESC_OCTET_LENGTH_PTR=1004;
+enum SQL_DESC_PRECISION=1005;
+enum SQL_DESC_SCALE=1006;
+enum SQL_DESC_DATETIME_INTERVAL_CODE=1007;
+enum SQL_DESC_NULLABLE=1008;
+enum SQL_DESC_INDICATOR_PTR=1009;
+enum SQL_DESC_DATA_PTR=1010;
+enum SQL_DESC_NAME=1011;
+enum SQL_DESC_UNNAMED=1012;
+enum SQL_DESC_OCTET_LENGTH=1013;
+
+enum SQL_DESCRIBE_PARAMETER=10002;
+
+enum SQL_DIAG_ALTER_DOMAIN=3;
+enum SQL_DIAG_ALTER_TABLE=4;
+enum SQL_DIAG_CALL=7;
+enum SQL_DIAG_CLASS_ORIGIN=8;
+enum SQL_DIAG_CONNECTION_NAME=10;
+enum SQL_DIAG_CREATE_ASSERTION=6;
+enum SQL_DIAG_CREATE_CHARACTER_SET=8;
+enum SQL_DIAG_CREATE_COLLATION=10;
+enum SQL_DIAG_CREATE_DOMAIN=23;
+enum SQL_DIAG_CREATE_INDEX=-1;
+enum SQL_DIAG_CREATE_SCHEMA=64;
+enum SQL_DIAG_CREATE_TABLE=77;
+enum SQL_DIAG_CREATE_TRANSLATION=79;
+enum SQL_DIAG_CREATE_VIEW=84;
+enum SQL_DIAG_DELETE_WHERE=19;
+enum SQL_DIAG_DROP_ASSERTION=24;
+enum SQL_DIAG_DROP_CHARACTER_SET=25;
+enum SQL_DIAG_DROP_COLLATION=26;
+enum SQL_DIAG_DROP_DOMAIN=27;
+enum SQL_DIAG_DROP_INDEX=(-2);
+enum SQL_DIAG_DROP_SCHEMA=31;
+enum SQL_DIAG_DROP_TABLE=32;
+enum SQL_DIAG_DROP_TRANSLATION=33;
+enum SQL_DIAG_DROP_VIEW=36;
+enum SQL_DIAG_DYNAMIC_DELETE_CURSOR=38;
+enum SQL_DIAG_DYNAMIC_FUNCTION=7;
+enum SQL_DIAG_DYNAMIC_FUNCTION_CODE=12;
+enum SQL_DIAG_DYNAMIC_UPDATE_CURSOR=81;
+enum SQL_DIAG_GRANT=48;
+enum SQL_DIAG_INSERT=50;
+enum SQL_DIAG_MESSAGE_TEXT=6;
+enum SQL_DIAG_NATIVE=5;
+enum SQL_DIAG_NUMBER=2;
+enum SQL_DIAG_RETURNCODE=1;
+enum SQL_DIAG_REVOKE=59;
+enum SQL_DIAG_ROW_COUNT=3;
+enum SQL_DIAG_SELECT_CURSOR=85;
+enum SQL_DIAG_SERVER_NAME=11;
+enum SQL_DIAG_SQLSTATE=4;
+enum SQL_DIAG_SUBCLASS_ORIGIN=9;
+enum SQL_DIAG_UNKNOWN_STATEMENT=0;
+enum SQL_DIAG_UPDATE_WHERE=82;
+
+enum SQL_FALSE=0;
+enum SQL_HANDLE_DBC=2;
+enum SQL_HANDLE_DESC=4;
+enum SQL_HANDLE_ENV=1;
+enum SQL_HANDLE_STMT=3;
+enum SQL_INSENSITIVE=1;
+enum SQL_MAX_CONCURRENT_ACTIVITIES=1;
+enum SQL_MAX_DRIVER_CONNECTIONS=0;
+enum SQL_MAX_IDENTIFIER_LEN=10005;
+enum SQL_MAXIMUM_CONCURRENT_ACTIVITIES=SQL_MAX_CONCURRENT_ACTIVITIES;
+enum SQL_MAXIMUM_DRIVER_CONNECTIONS=SQL_MAX_DRIVER_CONNECTIONS;
+enum SQL_MAXIMUM_IDENTIFIER_LENGTH=SQL_MAX_IDENTIFIER_LEN;
+enum SQL_NAMED=0;
+enum SQL_NO_DATA=100;
+enum SQL_NONSCROLLABLE=0;
+enum SQL_NULL_HANDLE=0L;
+enum SQL_NULL_HDESC=0;
+enum SQL_OJ_CAPABILITIES=115;
+enum SQL_OUTER_JOIN_CAPABILITIES=SQL_OJ_CAPABILITIES;
+enum SQL_PC_NON_PSEUDO=1;
+
+enum SQL_PRED_NONE=0;
+enum SQL_PRED_CHAR=1;
+enum SQL_PRED_BASIC=2;
+
+enum SQL_ROW_IDENTIFIER=1;
+enum SQL_SCROLLABLE=1;
+enum SQL_SENSITIVE=2;
+enum SQL_TIME_LEN=8;
+enum SQL_TIMESTAMP_LEN=19;
+enum SQL_TRUE=1;
+enum SQL_TYPE_DATE=91;
+enum SQL_TYPE_TIME=92;
+enum SQL_TYPE_TIMESTAMP=93;
+enum SQL_UNNAMED=1;
+enum SQL_UNSPECIFIED=0;
+enum SQL_XOPEN_CLI_YEAR=10000;
+}//#endif /* ODBCVER >= 0x0300 */
+
+extern (Windows) {
+    deprecated {
```

Comment:
Just confirming that this is intended to be `extern(Windows)` and not `extern(System)` and that these intended to be deprecated.